### PR TITLE
fix: improper spacing for Recurring Event in new booker

### LINKED ISSUE #9591: [CAL-1977] Improper spacing for Recurring Event in new booker
Input seems to be extra wide.
Also there seems to be some extra space below the input

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1977](https://linear.app/calcom/issue/CAL-1977/improper-spacing-for-recurring-event-in-new-booker)</sub>

### PR DESCRIPTION
**Pull request by kemilbeltre**
## What does this PR do?

I have removed the extra spacing and reduced the width of the InputNumber component in the new booker.

Fixes #9591 (issue)

Screenshot_20230617_205826

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Mandatory Tasks
- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by PeerRich** [comment]
do you have an "after" screenshot?

File: apps/web/components/booking/pages/AvailabilityPage.tsx [hunk 1]
```diff
@@ -258,14 +258,14 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
...
                       ? ([
-                          "Sunday",
-                          "Monday",
-                          "Tuesday",
-                          "Wednesday",
-                          "Thursday",
-                          "Friday",
-                          "Saturday",
-                        ].indexOf(profile.weekStart) as 0 | 1 | 2 | 3 | 4 | 5 | 6)
+                        "Sunday",
```
Comment (ujwalkumar1995) [inline apps/web/components/booking/pages/AvailabilityPage.tsx:261]: Hey @kemilbeltre 
Could you please check the lint issues?

**Comment by kemilbeltre** [comment]
> do you have an "after" screenshot?

The after screenshot is in the description.

File: apps/web/components/booking/pages/AvailabilityPage.tsx (hunk 1 above)
Comment (kemilbeltre) [reply apps/web/components/booking/pages/AvailabilityPage.tsx]: @ujwalkumar1995 done. Thanks

**Comment by anshuljangra003** [comment]
is this issue solved ?

**Comment by kemilbeltre** [comment]
> is this issue solved ?

Changes were very small

**Comment by JeroenReumkens** [comment]
Hey @kemilbeltre, 

The changes you did are actually for the old booker. If you want to run the new booker locally you need to add this to your env file:

```#Enables New booker for Embed only
NEW_BOOKER_ENABLED_FOR_EMBED=1

#Enables New booker for All but Embed requests
NEW_BOOKER_ENABLED_FOR_NON_EMBED=1
```

In that new booker the occurence input looks like this: 
CleanShot 2023-06-19 at 10 45 45@2x


Which I think looks okay?

**Comment by kemilbeltre** [comment]
> #Enables New booker for All but Embed requests
> NEW_BOOKER_ENABLED_FOR_NON_EMBED=1

Yes, but if you inspect that element you will notice extra spacing (right and button).

**Comment by kemilbeltre** [comment]
> Hey @kemilbeltre,
> 
> The changes you did are actually for the old booker. If you want to run the new booker locally you need to add this to your env file:
> 
> ```
> NEW_BOOKER_ENABLED_FOR_EMBED=1
> 
> #Enables New booker for All but Embed requests
> NEW_BOOKER_ENABLED_FOR_NON_EMBED=1
> ```
> 
> In that new booker the occurence input looks like this: CleanShot 2023-06-19 at 10 45 45@2x
> 
> Which I think looks okay?

@JeroenReumkens  could you confirm? Because I tested it and the extra spacing and width are still happening. Thanks

**Comment by JeroenReumkens** [comment]
> > Hey @kemilbeltre,
> > The changes you did are actually for the old booker. If you want to run the new booker locally you need to add this to your env file:
> > ```
> > NEW_BOOKER_ENABLED_FOR_EMBED=1
> > 
> > #Enables New booker for All but Embed requests
> > NEW_BOOKER_ENABLED_FOR_NON_EMBED=1
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > In that new booker the occurence input looks like this: CleanShot 2023-06-19 at 10 45 45@2x
> > Which I think looks okay?
> 
> @JeroenReumkens could you confirm? Because I tested it and the extra spacing and width are still happening. Thanks

Hi @kemilbeltre, the file you changed is part of the old booker. The multi event input for the new booker is located at `packages/features/bookings/components/event-meta/Occurences.tsx`. So removing the margin bottom should happen in the input in that file. Could you update this PR accordingly?

**Comment by kemilbeltre** [comment]
> > > Hey @kemilbeltre,
> > > The changes you did are actually for the old booker. If you want to run the new booker locally you need to add this to your env file:
> > > ```
> > > NEW_BOOKER_ENABLED_FOR_EMBED=1
> > > 
> > > #Enables New booker for All but Embed requests
> > > NEW_BOOKER_ENABLED_FOR_NON_EMBED=1
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > In that new booker the occurence input looks like this: CleanShot 2023-06-19 at 10 45 45@2x
> > > Which I think looks okay?
> > 
> > 
> > @JeroenReumkens could you confirm? Because I tested it and the extra spacing and width are still happening. Thanks
> 
> Hi @kemilbeltre, the file you changed is part of the old booker. The multi event input for the new booker is located at `packages/features/bookings/components/event-meta/Occurences.tsx`. So removing the margin bottom should happen in the input in that file. Could you update this PR accordingly?

@JeroenReumkens I think now is ready.  Thanks!

File: packages/features/bookings/components/event-meta/Occurences.tsx [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const EventOccurences = ({ event }: { event: PublicEvent }) => {
       {getRecurringFreq({ t, recurringEvent: event.recurringEvent })}
       <br />
       <Input
-        className="my-1 mr-3 inline-flex h-[26px] w-[46px] px-1 py-0"
+        className="my-1 mr-3 inline-flex h-[26px] w-[46px] px-1 py-0 mb-0"
```
Comment (kemilbeltre) [inline packages/features/bookings/components/event-meta/Occurences.tsx:64]: This is the only change I have made. Others are made by es-lint.

**Comment by PeerRich** [comment]
pretty sure this has been fixed in our new booking page. can you try our new booker?

**Pull request closed without merging**